src/styles/global.css

```diff
@@ -1,7 +1,7 @@
 @import url("https://fonts.googleapis.com/css2?family=Ubuntu&display=swap");
 
 html {
-  font-family: "Ubuntu", sans-serif;
+  font-family: "Ubuntu", sans-serif !important;
```

Comment:
no other style is overwriting this....not required

Reply:
No, it is required. I don't know why but the Roboto font is by default on everything. And writing important fixed this. See the difference. You will get it.
![image](https://user-images.githubusercontent.com/46858011/89637164-643c8200-d8c7-11ea-8053-614867c61148.png)
![image](https://user-images.githubusercontent.com/46858011/89637367-aebdfe80-d8c7-11ea-9e02-2dbaa541ee96.png)

Reply:
You can fix this issue by changing the font-family in global-styles.tsx to Ubuntu......there by default it has been kept sans-serif